cg/cli/set.py

```diff
@@ -59,7 +59,7 @@ def family(context, action, priority, panels, family_id):
 @click.option("--skip-lims", is_flag=True, help="Skip setting value in LIMS")
 @click.option("-y", "--yes", is_flag=True, help="Answer yes on all confirmations")
 @click.pass_context
-def samples(context, identifiers, kwargs, kip_lims, yes):
+def samples(context, identifiers, kwargs, skip_lims, yes):
```

Comment:
```suggestion
def samples(context, identifiers, kwargs, dry_run, yes):
```
This is dry_run now is it not?

Reply:
no it is about changing also in lims or not